vendor/bat-native-ads/src/bat/ads/internal/locale/country_code_util.cc

```diff
@@ -6,30 +6,30 @@
 #include "bat/ads/internal/locale/country_code_util.h"
 
 #include "brave/components/l10n/common/locale_util.h"
-#include "bat/ads/internal/locale/anonymous_country_codes.h"
-#include "bat/ads/internal/locale/large_anonymity_country_codes.h"
+#include "bat/ads/internal/locale/country_code_anonymity_set.h"
+#include "bat/ads/internal/locale/other_country_codes.h"
 
 namespace ads {
 namespace locale {
 
-bool HasLargeAnonymity(
+bool IsMemberOfAnonymitySet(
```

Comment:
good for now but thinking out loud: Right now the only variable by which we assign membership to the anonymity set is `country`. We'll likely be adding more soon, e.g. `channel` and `platform`. So `kCountryCodeAnonymitySet` and `kOtherCountryCodes` would give way to a more general approach.